libs/domain/content/link/link.component.ts

```diff
@@ -1,17 +1,26 @@
 import { resolve } from '@spryker-oryx/di';
 import { ContentMixin } from '@spryker-oryx/experience';
 import { LinkService } from '@spryker-oryx/site';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { LinkAppearance } from '@spryker-oryx/ui/link';
 import { computed, hydrate } from '@spryker-oryx/utilities';
 import { LitElement, TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { when } from 'lit/directives/when.js';
 import { ContentLinkContent, ContentLinkOptions } from './link.model';
+import { styles } from './link.styles';
 
 @hydrate()
 export class ContentLinkComponent extends ContentMixin<
   ContentLinkOptions,
   ContentLinkContent
 >(LitElement) {
+  static styles = styles;
+
+  @property({ reflect: true, type: Boolean })
+  protected dropdown = false;
```

Comment:
Why do you need a separate property if you have `appearance` option?

Reply:
It was for styling, but I see there's a comment below that addresses it.